cmd/genji/main.go

```diff
@@ -176,6 +176,48 @@ $ curl https://api.github.com/repos/genjidb/genji/issues | genji insert --db my.
 				return executeDump(c.Context, w, table, engine, dbPath)
 			},
 		},
+		{
+			Name:      "restore",
+			Usage:     "Restore a database from a file created by genji dump",
+			UsageText: `genji restore dumpFile dbPath`,
+			Description: `The restore command can restore a database from a text file.
+
+		$ genji restore -f dump.sql -e bolt my.db`,
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:    "file",
+					Aliases: []string{"f"},
+					Usage:   "name of the source file restoration.",
```

Comment:
Why do we need this option? cf https://github.com/genjidb/genji/issues/321#issuecomment-730380223